data_models/data_refinery_models/models/organism.py

```diff
@@ -0,0 +1,121 @@
+import requests
+from xml.etree import ElementTree
+from django.db import models
+from data_refinery_models.models.base_models import TimeTrackedModel
+
+# Import and set logger
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+NCBI_ROOT_URL = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/"
+ESEARCH_URL = NCBI_ROOT_URL + "esearch.fcgi"
+EFETCH_URL = NCBI_ROOT_URL + "efetch.fcgi"
+
+
+class UnscientifcNameError(BaseException):
+    pass
+
+
+class InvalidNCBITaxonomyId(BaseException):
+    pass
+
+
+def get_scientific_name(taxonomy_id: int):
```

Comment:
You're right. I started off with 3.5 so I didn't realize these were new in 3.5, I thought they were just a Python 3 thing. I'll update the README. I'll also add in some return values.